### ISSUE TITLE
About the experimental results and dataset

### ISSUE THREAD

**Issue by huangyc0321** [JXYin24/MCM#4]
I am interested in the method you proposed. However, after I ran your code, I found that there are a few datasets with results that are a little bit different than what you have in your paper, such as Thyroid, where I got a result of AUC-ROC: 0.9406 average AUC-PR: 0.5383; and Cardio, where I got a result of AVERAGE AUC-ROC: 0.8329 average AUC- PR: 0.5386.
My dataset was downloaded directly from https://odds.cs.stonybrook.edu/ in mat format, and the code and parameters were not modified. I would like to ask if I need to change some settings or parameters, and I hope you can add the dataset from the paper to your data directory for my reference, and preferably include the file in csv format, thank you very much!

**Comment by JXYin24** [comment]
Yes, we make a little modification of the parameters for different dataset, the abstract parameters are as follows:
![image](https://github.com/user-attachments/assets/5c73c042-17b6-4187-a4c2-c275f7544482)
